File: problem_thirty_four.js
//The coordinate point is inside or outside the circle find
var val1 = 1
var val2 =1
var basarddo = 4
var val3 = 10
var val4 = -14
function MyFunction(inp,inp2,inp3,inp4,bas){
    var value = Math.sqrt(((inp3-inp)**2 +(inp4-inp2)**2))
    
    if (value <= bas){
        console.log("The point is inside the circle.")
    }else{
        console.log("The point is not inside the circle.")
    }
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
}
MyFunction(val1,val2,val3,val4,basarddo)

File: problem_thirty_four.py
# The coordinate point is inside or outside the circle find
import math
x1 = int(input("Enter coordinate x1:"))
y1 = int(input("Enter coordinate y1:"))
x2 =int(input("Enter coordinate x2:"))
y2 = int(input("Enter coordinate y2:"))
z = int(input("Enter the basarddo:"))

def MyFunction(x1,x2,y1,y2,z):
    value = math.sqrt((x2-x1)**2 +(y2-y1)**2)
    if value <= z:
        print("The point is inside the circle.")
    else:
        print("The point is not inside the circle.")
MyFunction(x1,x2,y1,y2,z)    